web/src/message_reminder.ts

```diff
@@ -73,3 +123,34 @@ export function delete_reminder(reminder_id: number, success?: () => void): void
 export function get_count(): number {
     return reminders_by_id.size;
 }
+
+export function rerender_reminders_for_message(message_id: number): void {
+    const $rows = message_lists.all_rendered_row_for_message_id(message_id);
+    if ($rows.length === 0) {
+        return;
+    }
+
+    const message_reminders = reminders_by_message_id.get(message_id) ?? [];
+    if (message_reminders.length === 0) {
+        $rows.find(".message-reminders").remove();
+        return;
+    }
+
+    const rendered_message_reminders = render_message_reminders({
```

Comment:
An `_html` suffix is expected for variables containing HTML safe to insert in the DOM.